spec/lib/bank_account/account_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'bank_account'
+
+describe Bank::Account do
+  let(:account) {Bank::Account.new("check_it", 1000)}
+
+  describe "attributes" do
+    it "responds to :balance" do
+      expect(account).to respond_to :balance
+    end
+
+    it "responds to :id" do
+      expect(account).to respond_to :id
+    end
```

Comment:
I'd say the name of the example account, since that covers both that it has an attribute called `name` and that it's returning what you expect it to return.